manticore/ethereum/manticore.py

```diff
@@ -590,6 +590,15 @@ def solidity_create_contract(self, source_code, owner, name=None, contract_name=
                     else:
                         constructor_data = b''
 
+                    if balance != 0:
```

Comment:
@feliam Question: do we ever want to traverse such paths when creating contracts from solidity?

Shall we introduce a `force` keyword argument for such cases? If so, I would be worried we might never actually use it (hmm, but the tests should be good enough to be able to check e.g. things changed between solc versions?).

Reply:
We want to traverse *all* paths.